.github/workflows/Ci-raiwigets-python-typescript.yml

```diff
@@ -44,9 +44,10 @@ jobs:
       - name: Build Typescript
         run: yarn buildall
 
-      - if: ${{ matrix.operatingSystem == 'macos-latest' }}
```

Comment:
note: this libomp seems to be already installed by default and isn't needed actually based on logs